spec/policies/admin_policy_spec.rb

```diff
@@ -15,30 +15,30 @@
 
   describe 'For a member that is a part of a company' do
     let(:members_company) { Company.find_by_company_number('5562728336')}
-    subject { described_class.new(admin).authorized? }
+    subject { described_class.new(members_company).authorized? }
```

Comment:
On further review, you're absolutely right.
This doesn't make any sense.  Checking a _member_ and a _user_ is what is needed; a _company_ makes no sense.  (perhaps I was just thinking of a member associated with a different company.  Who knows.)